pom.xml

```diff
@@ -29,7 +29,7 @@
     </parent>
 
     <artifactId>chronicle-bytes</artifactId>
-    <version>2.24ea13-SNAPSHOT</version>
+    <version>2.24ea14-SNAPSHOT</version>
```

Comment:
?

Reply:
The current version is 2.24ea14-SNAPSHOT as refected in the chronicle-bom.
If you build version 2.24ea13-SNAPSHOT locally, you won't be using that version because the bom says to use 2.24ea14-SNAPSHOT

Reply:
It's actually weird that this is showing up in the diff, because according to my local copy it is already `2.24ea14-SNAPSHOT` on develop

Reply:
Maybe just need to rebase?